_attributes/common-attributes.adoc

```diff
@@ -42,6 +42,7 @@ endif::openshift-origin[]
 :insights-advisor-url: link:https://console.redhat.com/openshift/insights/advisor/[Insights Advisor]
 :hybrid-console: Red Hat Hybrid Cloud Console
 :hybrid-console-second: Hybrid Cloud Console
+:product-registry: OpenShift image registry
```

Comment:
Suggestion: would you mind adding this to the `attributes-midcroshift.adoc` and `attributes-openshift-dedicated.adoc` files too please? Some registry content is in the SD docs an may also be in the MicroShift docs as well :)

Reply:
Done!